hmftools.wdl

```diff
@@ -0,0 +1,927 @@
+        File tumorBafVcf = "~{outputDir}/~{tumorName}.amber.baf.vcf.gz"
+        File tumorBafVcfIndex = "~{outputDir}/~{tumorName}.amber.baf.vcf.gz.tbi"
+        File tumorContaminationVcf = "~{outputDir}/~{tumorName}.amber.contamination.vcf.gz"
+        File tumorContaminationVcfIndex = "~{outputDir}/~{tumorName}.amber.contamination.vcf.gz.tbi"
+        File tumorContaminationTsv = "~{outputDir}/~{tumorName}.amber.contamination.tsv"
+        File tumorQc = "~{outputDir}/~{tumorName}.amber.qc"
+        File normalSnpVcf = "~{outputDir}/~{normalName}.amber.snp.vcf.gz"
+        File normalSnpVcfIndex = "~{outputDir}/~{normalName}.amber.snp.vcf.gz.tbi"
+        Array[File] outputs = [version, tumorBafPcf, tumorBafTsv, tumorBafVcf, tumorBafVcfIndex, 
+            tumorContaminationVcf, tumorContaminationVcfIndex, tumorContaminationTsv, tumorQc, 
+            normalSnpVcf, normalSnpVcfIndex]
+    }
+
+    runtime {
+        memory: memory
+        time_minutes: timeMinutes # !UnknownRuntimeKey
+        docker: dockerImage
+        cpu: threads
+    }
+
+    parameter_meta {
+        normalName: {description: "the name of the normal sample.", category: "required"}
+        normalBam: {description: "The normal BAM file.", category: "required"}
+        normalBamIndex: {description: "The index for the normal BAM file.", category: "required"}
+        tumorName: {description: "The name of the tumor sample.", category: "required"}
+        tumorBam: {description: "The tumor BAM file.", category: "required"}
+        tumorBamIndex: {description: "The index for the tumor BAM file.", category: "required"}
+        outputDir: {description: "The path to the output directory.", category: "common"}
+        loci: {description: "A VCF file containing likely heterozygous sites.", category: "required"}
+        referenceFasta: {description: "The reference fasta file.", category: "required"}
+        referenceFastaDict: {description: "The sequence dictionary associated with the reference fasta file.",
+                             category: "required"}
+        referenceFastaFai: {description: "The index for the reference fasta file.", category: "required"}
+        threads: {description: "The number of threads the program will use.", category: "advanced"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        javaXmx: {description: "The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.",
+                  category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task Cobalt {
+    input {
+        String normalName
+        File normalBam
```

Comment:
Again normalBam instead of referenceBam. I assume this is for consistency with other tools? Otherwise it might be a bit confusing as we normally follow the command line flags quite closely.